backend/src/models/student.js

```diff
@@ -0,0 +1,75 @@
+const { Model } = require('objection')
+const tableStudents = 'students'
+
+class Student extends Model {
+  static get tableName () {
+    return tableStudents
+  }
+
+  static get idColumn () {
+    return ['StudentID']
+  }
+
+  fullName () {
+    return `${this.firstName} ${this.lastName}`
+  }
+
+  static get relationMappings () {
+    const NativeLanguage = require('./nativeLanguage')
+    const Status = require('./status')
+    const StatusUpdate = require('./statusUpdate')
+
+    return {
+      nativeLanguage: {
+        relation: Model.HasOneRelation,
+        modelClass: NativeLanguage,
+        join: {
+          from: 'students.NativeLanguageID',
+          to: 'nativeLanguages.NativeLanguageID'
+        }
+      },
+
+      statusID: {
```

Comment:
```suggestion
      status: {
```
We are retrieving the `Status` at a given `StatusID`.